environment/prod/deployment/prod/apps/eskrzypacz/.env.prod

```diff
@@ -0,0 +1,16 @@
+APP_NAME="ESkrzypacz Keating"
+ENVIRONMENT=prod
+APP_DEBUG=false
+
+COMPOSE_PROJECT_NAME=eskrzypacz
+TRAEFIK_ENABLED=true
+KEATING_HOST_NAME=
+APP_URL=https://${KEATING_HOST_NAME}/
+TRAEFIK_ROUTER_RULE=
+
+DB_DATABASE=eskrzypacz
```

Comment:
How creating databases and users with passwords will be handled?

Reply:
I hope I understood the question correctly. 
Each application has in the .env variables needed for the database: user, password and database name. At the moment I want to run the application, maybe later I will be able to automate the process of creating databases more.
I want to start the first deployment automatically through the workflow - a container will be created and probably one indicated database, then I will create additional databases for the remaining 3 applications manually via the console and then run subsequent deployments for individual applications. First, I also want to see if this solution will work.

Reply:
So you can test it before final approve. 
Also it will be good to write instructions (I think in this repo?) how to work with this, how to add new app instance/user.

Reply:
Of course, before the final approve, I will test this PR and also add the entire instructions to readme.md. But now I wanted to check this PR initially. At the moment I am still waiting for information about domains.